corehq/apps/domain/tests/test_delete_domain.py

```diff
@@ -251,12 +251,14 @@ def test_active_subscription_credits_transferred_to_account(self):
 
         self.domain.delete()
 
+        # Check that get_credits_by_subscription_and_features does not return the old deactivated credit line
         subscription_credits = CreditLine.get_credits_by_subscription_and_features(
             self.current_subscription,
             feature_type=FeatureType.SMS,
         )
-        self.assertEqual(len(subscription_credits), 1)
-        self.assertEqual(subscription_credits[0].balance, Decimal('0.0000'))
+        self.assertEqual(len(subscription_credits), 0)
```

Comment:
@nickpell I changed `test_active_subscription_credits_transferred_to_account` because lines 258 and 259 were failing. Now, after a domain is deleted and its subscription's credits are deactivated, the old credits won't show up (but the credits that were transferred to the account level will still show).

Let me know if this is an acceptable workflow. If we want the old credits to still show up in the subscription's credit line list, we can set the credit balance to `0` and not keep `is_active = True`, and that should keep the same workflow we had before